master_data/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.1.3 on 2024-11-26 20:21
+
+import master_data.utility.media_utility
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='VendorCategory',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=100, unique=True)),
+                ('description', models.TextField()),
+                ('icon', models.ImageField(upload_to=master_data.utility.media_utility.vendor_category_icon_paths)),
+                ('deleted_flag', models.BooleanField(default=False)),
```

Comment:
Instead of a flag, what if we store `deleted_at` so that we don't need to query `updated_at` as well when we have to double check when this record is deleted?

Reply:
IMO, the purpose of `updated_at` is to record a timestamp for all activities made in a particular data row. Therefore, replacing it with `deleted_at` is not a solution

Reply:
I mean, we keep the `updated_at` as well but replace `deleted_flag` with `deleted_at`. We set `deleted_at` when we delete a record; if it's not, the value is `NULL`. But both approaches are fine to me.

Reply:
Good point.

However, let's try the simple version using this boolean field first. Maybe later we find more concrete use case that require us to use DateTimeField instead of the BooleanField.